docs/src/test-api/class-testconfig.md

```diff
@@ -321,6 +321,22 @@ This path will serve as the base directory for each test file snapshot directory
 ## property: TestConfig.snapshotPathTemplate = %%-test-config-snapshot-path-template-%%
 * since: v1.28
 
+## property: TestConfig.populateGitInfo
+* since: v1.50
```

Comment:
```suggestion
* since: v1.51
```
We are going to cut 1.50 branch in the next few days and would like to push this feature to the next release.